core/src/main/java/com/ultramixer/jarbundler/JarBundler.java

```diff
@@ -502,6 +502,9 @@ public void setContentSize(String s) {
      * @param b True sets 'JavaX' dictionary key instead of 'Java' key
      */
     public void setUseJavaXKey(boolean b) {
+		if (b && (bundleProperties.getJavaVersion() >= 1.7)) {
+			throw new BuildException("Setting usejavaxkey is useless if jvmversion is at least 1.7, because then the Oracle PList format is used");
```

Comment:
That's not correct. It's still an option. See #12 and maybe #8 for a lenghty discussion about that.

Will not merge this PR with these changes here, sorry...

Reply:
Fine. Just my clumsiness.